__test__/test_helpers.js

```diff
@@ -101,18 +100,18 @@ function getGqlOperationText(op) {
 }
 
 export async function runGql(operation, vars, user) {
-  const operationText = getGqlOperationText(operation) || operation;
+  const source = getGqlOperationText(operation) || operation;
   const rootValue = {};
-  const context = getContext({ user });
-  const result = await graphql(
-    mySchema,
-    operationText,
+  const contextValue = getContext({ user });
+  const result = await graphql({
```

Comment:
Changed arguments to match new signature.

Reply:
What is the new signature in reference to?

Reply:
the `graphql` function takes different parameters in the updated version compared to the older version